source/mslite-metadata.json

```diff
@@ -0,0 +1,2140 @@
+        { "name": "aspect_ratios", "type": "float32[]" },
+        { "name": "variances", "type": "float32[]" },
+        { "name": "image_size_w", "type": "int32", "default": 0 },
+        { "name": "image_size_h", "type": "int32", "default": 0 },
+        { "name": "step_w", "type": "float32", "default": 0.0 },
+        { "name": "step_h", "type": "float32", "default": 0.0 },
+        { "name": "clip", "type": "boolean", "default": true },
+        { "name": "flip", "type": "boolean", "default": true },
+        { "name": "offset", "type": "float32", "default": 0.0 }
+      ]
+    }
+  },
+  {
+    "name": "Proposal",
+    "schema": {
+      "attributes": [
+        { "name": "feat_stride", "type": "float32", "default": 0.0 },
+        { "name": "base_size", "type": "float32", "default": 0.0 },
+        { "name": "min_size", "type": "float32", "default": 0.0 },
+        { "name": "ratio", "type": "float32[]" },
+        { "name": "scale", "type": "float32[]" },
+        { "name": "pre_nms_topn", "type": "int32", "default": 0 },
+        { "name": "post_nms_topn", "type": "int32", "default": 0 },
+        { "name": "nms_thresh", "type": "float32", "default": 0.0 }
+      ]
+    }
+  },
+  {
+    "name": "QuantDTypeCast",
+    "schema": {
+      "attributes": [
+        { "name": "srcT", "type": "int32", "default": 0 },
+        { "name": "dstT", "type": "int32", "default": 0 }
+      ]
+    }
+  },
+  {
+    "name": "Range",
+    "schema": {
+      "outputs": [
+        { "name": "output" }
+      ],
+      "attributes": [
+        { "name": "dType", "type": "int32", "default": 0 },
+        { "name": "start", "type": "int32", "default": 0 },
+        { "name": "limit", "type": "int32", "default": 0 },
+        { "name": "delta", "type": "int32", "default": 0 }
+      ]
+    }
+  },
+  {
+    "name": "Rank",
+    "schema": {
+      "inputs": [
+        { "name": "input" }
+      ],
+      "outputs": [
+        { "name": "output" }
+      ]
+    }
+  },
+  {
+    "name": "RealDiv",
+    "schema": {
+      "inputs": [
+        { "name": "x"},
+        { "name": "y"}
+      ],
+      "outputs": [
+        { "name": "z" }
+      ]
+    }
+  },
+  {
+    "name": "Reduce",
+    "schema": {
+      "inputs": [
+        { "name": "input" }
+      ],
+      "outputs": [
+        { "name": "output" }
+      ],
+      "attributes": [
+        { "name": "axes", "type": "int32[]"},
+        { "name": "keepDims", "type": "int32", "default": 0 },
+        { "name": "mode", "type": "ReduceMode"},
+        { "name": "reduceToEnd", "type": "boolean", "default": false },
+        { "name": "coeff", "type": "float32", "default": 1.0 }
+      ]
+    }
+  },
+  {
+    "name": "Reshape",
+    "schema": {
+      "inputs": [
+        { "name": "input" },
+        { "name": "shape" }
+      ],
+      "outputs": [
+        { "name": "output" }
+      ],
+      "attributes": [
+        { "name": "format", "type": "Format" },
+        { "name": "shape", "type": "int32[]" }
```

Comment:
According to `ops.fbs` this should be `int64[]`? e.g. `long`=`int64` and `int`=`int32`.

Reply:
> According to `ops.fbs` this should be `int64[]`? e.g. `long`=`int64` and `int`=`int32`.

YES